### PR TITLE
feat: adding changelog file #39 and release script

### PR DESCRIPTION
**Pull request by saiSunkari19**
## Description

Adding change log file 

---

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Added Features

- [ ]

---

## Modified Features

- [ ]

---

## Test Cases

- [ ] Test A
- [ ] Test B

---

## Change Log

---

## Documentation

---

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
---

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# This workflow is useful if you want to automate the process of:
+#
+# a) Creating a new prelease when you push a new tag with a "v" prefix (version).
```
Comment (3eyed-raven) [inline .github/release.yml:3]: pre-release?

Comment (saiSunkari19) [reply .github/release.yml:3]: I didn’t get you.

Comment (3eyed-raven) [reply .github/release.yml:3]: There is a typo error in the sentence. prelease --> pre-release

Comment (3eyed-raven) [reply .github/release.yml:3]: also if you can change all usage of prerelease --> pre-release that will be great.

Comment (saiSunkari19) [reply .github/release.yml:3]: Sure

**Review by 3eyed-raven — APPROVED** [review]

**Pull request merged**